synapse/cortex.py

```diff
@@ -1538,10 +1538,10 @@ def _initCorePerms(self):
              'desc': 'Controls access to deleting storm packages.'},
 
             {'perm': ('storm', 'asroot', 'cmd', '<cmdname>'), 'gate': 'cortex',
-            'desc': 'Controls running storm commands requiring root privileges.',
+            'desc': 'Deprecated. Controls running storm commands requiring root privileges.',
```

Comment:
Could we update these to "Deprecated. Please use..." to make it even less likely that someone will begin using them because they read and understand what they do ;)